backend/app/controllers/sessions_controller.rb

```
@@ -39,29 +39,55 @@ def destroy
-    provider = params[:provider]
+    provider = auth&.provider
     user = User.from_omniauth(auth)
 
     if user
       token = JsonWebToken.encode(user_id: user.id)
 
-      if provider == "youtube"
-        # Handle YouTube platform connection
-        youtube_connection = user.platform_connections.find_or_initialize_by(platform: 'youtube')
+      # Normalize fields defensively
+      raw_expires_at = (auth.credentials.expires_at rescue nil)
```

RuboCop (1.76.1)

[convention] Avoid using `rescue` in its modifier form.

(Style/RescueModifier)